packages/js/product-editor/typings/index.d.ts

```diff
@@ -25,9 +25,21 @@ declare module '@wordpress/core-data' {
 		name: string,
 		id: number | string,
 		options?: { enabled: boolean }
-	): { record: T, editedRecord: T, isResolving: boolean, hasResolved: boolean };
+	): {
+		record: T;
+		editedRecord: T;
+		isResolving: boolean;
+		hasResolved: boolean;
+	};
 }
 declare module '@wordpress/keyboard-shortcuts' {
-	function useShortcut(name: string, callback: (event: KeyboardEvent) => void): void;
+	function useShortcut(
+		name: string,
+		callback: ( event: KeyboardEvent ) => void
+	): void;
```

Comment:
Above changes are formatting.
I only added the below item for the router module.